### PR TITLE
JSON is no longer required ( #10 )

### LINKED ISSUE #10: Optional JSON
Currently the player only works if you provide a query string for both mp4 json files. The json should made to be optional and just don't render the chat if it is not supplied. The input in the form needs to be made as not required as well.

### PR DESCRIPTION
**Pull request by ERmilburn02**
Changes to the way comments are loaded means that a JSON file is no longer required as an argument. (Closes #10 )

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -108,12 +112,14 @@ const r = () => {
...
     });
 
-    fetch(json).then((res) => {
-        res.json().then((data) => {
-            comments = data.comments;
-            renderChat();
+    if (hasJSON) {
+        fetch(json).then((res) => {
+            res.json().then((data) => {
+                comments = data.comments;
```
Comment (arya-s) [inline script.js:118]: nit: I'd say ideally throw an error if comments is undefined.

**Review by arya-s — APPROVED** [review]
LGTM other than the nit comment

File: script.js (hunk 1 above)
Comment (ERmilburn02) [reply script.js:118]: This is Evan's existing code, I just wrapped it in the `if (hasJSON)`

Comment (arya-s) [reply script.js:118]: Fair enough.

**Pull request merged**